app/src/components/annotation/AnnotationDetailsContent.tsx

```diff
@@ -0,0 +1,89 @@
+import { Flex, Text, View } from "@phoenix/components";
+import { AnnotationColorSwatch } from "@phoenix/components/annotation/AnnotationColorSwatch";
+import { Truncate } from "@phoenix/components/utility/Truncate";
+import { floatFormatter } from "@phoenix/utils/numberFormatUtils";
+
+import type { Annotation } from "./types";
+
+/**
+ * A component that displays the details of a particular annotatio
+ */
+export function AnnotationDetailsContent({
```

Comment:
this is just the previous tooltip content moved into it's own component to be re-used